### PR TITLE
add crypto token

### PR DESCRIPTION
**Pull request by vhdirk**
## PR Checklist

Please check if your PR fulfills the following requirements:

-   [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0-beta.4/)
-   [x] Tests for the changes have been added (for bug fixes / features)
-   [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

-   [ ] Bugfix
-   [x] Feature
-   [ ] Refactoring (no functional changes, no api changes)
-   [ ] Other... Please describe:

## What is the current behavior?

This PR introduces a token for the [Web Crypto API](https://developer.mozilla.org/en-US/docs/Web/API/Web_Crypto_API)

Issue Number: N/A

## Does this PR introduce a breaking change?

-   [ ] Yes
-   [x] No

### CODE REVIEW

**Review by MarsiBarsi — APPROVED** [review]
Hey! Thanks for your contribution :)

File: projects/common/src/tokens/crypto.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import {inject, InjectionToken} from '@angular/core';
+import {WINDOW} from './window';
+
+export const CRYPTO = new InjectionToken<Crypto>(
+    'An abstraction over window.crypto object',
+    {
+        factory: () => inject(WINDOW).localStorage,
```
Comment (waterplea) [inline projects/common/src/tokens/crypto.ts:7]: You forgot to change this 🙂

**Review by waterplea — CHANGES_REQUESTED** [review]
Looks like checks are not running for external contributions. We'll fix that, in the meantime please check my comment 😉

File: projects/common/src/tokens/crypto.ts (hunk 1 above)
Comment (vhdirk) [reply projects/common/src/tokens/crypto.ts]: Woops, indeed. copy/pasted too quickly :)

Comment (vhdirk) [reply projects/common/src/tokens/crypto.ts]: Not sure of github sends notifications for force-pushes. Updated, though.

**Review by waterplea — APPROVED** [review]

**Pull request merged**